src/jarabe/screenshotpanel/gui.py

```diff
@@ -0,0 +1,341 @@
+# Copyright (C) 2008 One Laptop Per Child
```

Comment:
Is this a new file, and if so why are you assigning copyright to One Laptop per Child yet dating it 2008?  I'm fine with you assigning copyright to us, but if so please use the current year.